uberfire-project/uberfire-project-api/src/main/java/org/guvnor/common/services/project/context/WorkspaceProjectContextChangeEvent.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2013 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.guvnor.common.services.project.context;
+
+import org.guvnor.common.services.project.model.Module;
+import org.guvnor.common.services.project.model.Package;
+import org.guvnor.common.services.project.model.WorkspaceProject;
+import org.guvnor.structure.organizationalunit.OrganizationalUnit;
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+/**
+ * An event raised when the Project Context changes
+ */
+@Portable
```

Comment:
Since the WorkspaceProjectContext is only used on the client now, this probably doesn't need to be portable. In fact, we should probably move it to the client and confirm that is isn't referenced in backend code.